src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx

```diff
@@ -172,7 +187,7 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
       state.status.key,
       state.check.checked,
       onUpdate,
-      openEmailPreviewModal,
+      getEmailTemplate,
     ]
```

Comment:
Missing dependency

Reply:
weird, the missing dependency is there...

Reply:
state.status.key Isn't...

Reply:
it was there earlier